### ISSUE TITLE
Search Console Issues

### ISSUE THREAD

**Issue by MaksymOsovitnii** [Blueshoe/website#138]

**Comment by MaksymOsovitnii** [comment]
@TabeaHe , I have created this task just to keep track of the progress and give you updates. Please, let me know, when I will be invited to Search Console ( maximosovitny@gmail.com )

**Comment by TabeaHe** [comment]
@MaksymOsovitnii Hannes added you to the Search Console for blueshoe.io. Do you have access?

**Comment by MaksymOsovitnii** [comment]
@TabeaHe , yes, now I have it and will check it. Will be back to you.

**Comment by MaksymOsovitnii** [comment]
@TabeaHe , all issues are fixes, all pages are indexed without errors from sitemap I have submitted. https://search.google.com/u/0/search-console/index?resource_id=sc-domain:blueshoe.io&pages=SITEMAP&sitemap=https:%2F%2Fwww.blueshoe.io%2Fsitemap.xml

Should I do the same for DE version? Can I have access to search console of DE version?

**Comment by TabeaHe** [comment]
@MaksymOsovitnii I understand that all pages included in the sitemap are indexed without issues. What is the rest?
![image](https://github.com/Blueshoe/website/assets/106660869/05863199-67bd-41f8-981d-443da2717a64)

I will come back to you because of the German website.

**Comment by TabeaHe** [comment]
@MaksymOsovitnii Since yesterday you should have access to the Search Console account of blueshoe.de . Could you solve the issues there as well, please?

**Comment by MaksymOsovitnii** [comment]
@TabeaHe , deployed DE version too and launched the same check in Search Console. Also, if possible, provide me with access to Google Analytics for both version. Thanks.

+ There should be no more issues with trailing slash in the new data in Google Analytics for both versions

**Comment by Truthans** [comment]
@MaksymOsovitnii I just added you to GA. Let me know in case you need more rights.

**Comment by MaksymOsovitnii** [comment]
@Truthans , thank you

**Comment by TabeaHe** [comment]
@MaksymOsovitnii What's the status here concerning the search console and the trailing slash issue is GA?

**Comment by MaksymOsovitnii** [comment]
@TabeaHe , I think all is fixed. Now website works with trailing slash and without. Most of  issues are connected not to the main website (like these 83 issues) https://share.cleanshot.com/xvg2cQjM . Also some are under review now, will see the results: https://share.cleanshot.com/Wlw1fB82

**Comment by TabeaHe** [comment]
@MaksymOsovitnii How do we get one single analysis on Google Analytics for https://www.blueshoe.io/our-services/api-development-agency/ and https://www.blueshoe.io/our-services/api-development-agency ? The problem occurs for example here 
![image](https://github.com/Blueshoe/website/assets/106660869/f7b9d32b-72b5-4df1-ba25-8ca4f8ff5fcc)

**Comment by MaksymOsovitnii** [comment]
@TabeaHe , you are right, now to prevent any external traffic from urls without slash I will handle this automatically. Please try to visit this page https://www.blueshoe.io/our-services/cloud-application-development and this one https://www.blueshoe.de/leistungen/python-django-agentur. As you see I redirect to the url with / in the end so all pages in Google Analytics and their traffic starting from tomorrow should have the same url. It should solve the issue. Redirect is done with 308 status code, which is is correct from SEO point of view.

**Comment by TabeaHe** [comment]
@MaksymOsovitnii The two links are redirected to the links with "/". I'll have a look at Google Analytics next week and check how the data is shown there.

**Comment by MaksymOsovitnii** [comment]
@TabeaHe , ok, let me know if you will find any issues
